cmd/common.go

```diff
@@ -101,6 +101,14 @@ func syncMain(ctx context.Context, filenames []string, dry bool, parallelism,
 	var kongClient *kong.Client
 	mode := getMode(targetContent)
 	if mode == modeKonnect {
+		if targetContent.Workspace != "" {
+			return fmt.Errorf("_workspace set in config file.\n" +
+				"Workspaces are not supported in Konnect. " +
+				"Please remove '_workspace: foo' from your configuration and try again")
```

Comment:
'foo' seems to be a copy-paste error. Could you please update the error message to include the workspace that the user has provided?

Reply:
done
```
$ ./deck sync --konnect-token $TOKEN_DEV
Error: _workspace set in config file.
Workspaces are not supported in Konnect. Please remove '_workspace: test' from your configuration and try again
```